tests/slow/test_pyflexplot/test_plots/ref_cosmo-1_affected_area.py

```diff
@@ -982,32 +982,32 @@
                         "xlabel": "",
                         "xlim": (0.19550009091695134, 1.8204999319712316),
                         "xmajorticklabels": [
-                            "Text(0, 0, '')",
-                            "Text(0, 0, '')",
-                            "Text(0, 0, '')",
-                            "Text(0, 0, '')",
-                            "Text(0, 0, '')",
-                            "Text(0, 0, '')",
-                            "Text(0, 0, '')",
-                            "Text(0, 0, '')",
-                            "Text(0, 0, '')",
-                            "Text(0, 0, '')",
-                            "Text(0, 0, '')",
+                            "Text(0.0, 0, '0.0')",
```

Comment:
Might be worth removing code duplication by defining identical blocks only once. This would also reduce the amount of maintenance work and the size of the diffs in further PRs... (this comment is valid for all files changed in `tests/slow`)

Reply:
All files in tests/slow are dictionaries of references for some unit tests to compare the results with the generated output. These are automatically updated with help of a command.